management/appointments/index.php

```diff
@@ -8,115 +8,127 @@
+				<nav class="dcf-breadcrumbs" id="dcf-breadcrumbs" role="navigation" aria-label="breadcrumbs">
+					<!-- TemplateBeginEditable name="breadcrumbs" -->
+					<ol>
+						<li><a href="https://www.unl.edu/">Nebraska</a></li>
+						<li><a href="/">VITA Lincoln</a></li>
+						<li><a href="/management">Management</a></li>
+						<li><span aria-current="page">Appointment Management</span></li>
+					</ol>
 					<!-- TemplateEndEditable -->
-					<?php wdnInclude("/wdn/templates_4.1/includes/navigation-addons.html"); ?>
 				</nav>
 			</div>
+			<header class="dcf-page-title" id="dcf-page-title">
+				<!-- TemplateBeginEditable name="pagetitle" -->
+				<h1>Appointment Management</h1>
+				<!-- TemplateEndEditable -->
+			</header>
+			<!-- TemplateBeginEditable name="herogroup1" -->
+			<!-- TemplateEndEditable -->
 		</div>
-		<div class="wdn-menu-trigger wdn-content-slide">
-			<label for="wdn_menu_toggle" class="wdn-icon-menu">Menu</label>
-			<?php wdnInclude("/wdn/templates_4.1/includes/share.html"); ?>
+		<!-- TemplateBeginEditable name="herogroup2" -->
+		<div class="dcf-hero-group-2">
 		</div>
-		<main id="wdn_content_wrapper" role="main" class="wdn-content-slide" tabindex="-1">
-			<div id="maincontent" class="wdn-main">
-				<div id="pagetitle">
-					<!-- TemplateBeginEditable name="pagetitle" -->
-					<h1>Appointment Management</h1>
-					<!-- TemplateEndEditable -->
-				</div>
-				<!-- TemplateBeginEditable name="maincontentarea" -->
-				<div id="appointmentsApp" class="wdn-band" appointments></div>
-				<!-- TemplateEndEditable -->
-			</div>
-		</main>
-		<footer id="footer" role="contentinfo" class="wdn-content-slide">
-			<div id="wdn_optional_footer" class="wdn-band wdn-footer-optional">
-				<div class="wdn-inner-wrapper">
-					<!-- TemplateBeginEditable name="optionalfooter" -->
-					<!-- TemplateEndEditable -->
-				</div>
-			</div>
-			<div id="wdn_local_footer" class="wdn-band wdn-footer-local">
-				<div class="wdn-inner-wrapper">
-					<!-- TemplateBeginEditable name="contactinfo" -->
-					<?php include "$root/sharedcode/localFooter.html"; ?>
-					<!-- TemplateEndEditable -->
-					<!-- TemplateBeginEditable name="leftcollinks" -->
-					<!-- TemplateEndEditable -->
-				</div>
-			</div>
-			<div id="wdn_global_footer" class="wdn-band wdn-footer-global">
-				<div class="wdn-inner-wrapper">
-					<?php wdnInclude("/wdn/templates_4.1/includes/globalfooter.html"); ?>
-				</div>
-			</div>
-		</footer>
-		<?php wdnInclude("/wdn/templates_4.1/includes/noscript.html"); ?>
+		<!-- TemplateEndEditable -->
 	</div>
-	<?php wdnInclude("/wdn/templates_4.1/includes/body_scripts.html"); ?>
-	<?php require_once "$root/server/global_includes.php"; ?>
-	<script src="/dist/management/appointments/appointments.js"></script>
+	<div class="dcf-main-content dcf-wrapper">
+		<!-- TemplateBeginEditable name="maincontentarea" -->
+		<div id="appointmentsApp" class="wdn-band" appointments></div>
```

Comment:
There's still a `wdn-band` on this one. Should that change to `dcf-wrapper`? Or be removed entirely?

Reply:
As a matter of fact, there's quite a few instances of `wdn` classes being used. I used the following regex across the entire codebase: `class=".*wdn.*"` and found 61 instances of `wdn` classes--should some of these be changed/removed? Or are those 61 instances good?

Reply:
I've addressed a lot of these. Could you take a look at the last few? I couldn't find any corresponding `dcf` classes for them.